Apps/WebClient/src/ClientApp/src/views/reports.vue

```diff
@@ -312,15 +312,15 @@ export default class ReportsView extends Vue {
                             <b-row>
                                 <b-col class="col-12 col-lg-4 pt-3">
                                     <label for="start-date">From</label>
-                                    <DatePickerComponent
+                                    <hg-date-picker
```

Comment:
why this change?

Reply:
The automated generated stub from shallowmount cases this as ````datepickercomponent-stub```. The change make sit easier to stub since the name is not changed. This is also in line with other html component changes like the hg-button and icon